locales/mr-IN.ftl

```diff
@@ -0,0 +1,38 @@
+Title = Firefox Quantum रिलीझ मोहीम
+
+Project-name = Firefox Quantum रिलीझ
+
+main-title = Firefox परत आले आहे!
+
+main-description = आम्हाला नवीन Firefox बद्दल जगात प्रसारण करण्यास मदत करा! मंगळवार 14 ला,14:00 UTC वाजता #NewFirefox वापरून आपल्याला त्यातील काय आवडते त्या विषयी ट्विट करा किंवा
+Firefox चे ट्विट परत रिट्विट करा.
+
+thunderclap-cta = Firefox Quantum चे समर्थन करा
+
+thunderclap-link = https://www.thunderclap.it/projects/64595-firefox-quantum-launch-day
+
+official-tweet-cta = अधिकृत ट्विट रिट्विट करा
+
+Tweet-1 = आज #NewFirefox आले आहे आणि तेही पूर्वीपेक्षा दुप्पट जलद :boom: डाउनलोड करा:https://mzl.la/newfirefox
+
+Tweet-2 = #NewFirefox 🦊 एका ___ पेक्षा जलद आहे. धन्यवाद @Firefox!
+
+Tweet-3 = नवीन @Firefox आले आहे आणि आता अधिक जलद आहे. #NewFirefox येथे मिळवा: https://mzl.la/newfirefox
+
+own-tweet-cta = स्वतःचे ट्विट करा
+
+Hashtag = #NewFirefox
+
+github = Github
+
+Legal = कायदेविषयक
+
+License = परवाना
+
+Guidelines = मार्गदर्शक तत्त्वे
+
+Mozilla-description = Mozilla एक जागतिक विना-नफा आहे जो आपल्याला आपल्या ऑनलाइन अनुभवावर नियंत्रण ठेवण्यासाठी आणि
```

Comment:
please keep the same formatting as in the en-US file.

Reply:
Thanks , Please review again.